src/PolyhedralGeometry/Polyhedron/constructors.jl

```diff
@@ -45,6 +45,29 @@ julia> b = [1, 1, 0, 0];
 julia> Polyhedron(A,b)
 A polyhedron in ambient dimension 2
 ```
+"""
+Polyhedron{T}(A::AnyVecOrMat, b::AbstractVector) where T<:scalar_types = Polyhedron{T}((A, b))
+
+Polyhedron{T}(A::AbstractVector{>:AbstractVector}, b::Any) where T<:scalar_types = Polyhedron{T}([A], [b])
```

Comment:
It might look like that, but actually that syntax is used to write the complement, so that `AbstractVector` is supposed to comtain anything but another kind of `AbstractVector`.

Reply:
No, now I am very sure that is not the case. Not sure why i thought this was the case. I will refine the handling here.